services/core-api/app/api/now_applications/transmogrify_now.py

```diff
@@ -10,9 +10,9 @@
 
```

Comment:
All auto-formatting below. I just changed these "ton" values to match the fixed code values in the DB.